lib/wibox/widget/slider.lua

```diff
@@ -349,6 +362,10 @@ function slider:draw(_, cr, width, height)
         or beautiful.slider_handle_border_width
         or properties.handle_border_width or 0
 
+    local handle_cursor = self._private.handle_cursor
+        or beautiful.slider_handle_cursor
+        or properties.handle_cursor or "fleur"
```

Comment:
i think you don't need the latest `or "fleur"` here, since it's already defined on line 286 as properties.handle_cursor

Reply:
fixed!